Jenkinsfile

```diff
@@ -0,0 +1,173 @@
+    }
+
+    stages {
+
+        stage('Prepare') {
+            steps {
+                script {
+                    if (env.CHANGE_ID != null) {
+                        def json = sh (script: "curl -s https://api.github.com/repos/crossplaneio/stack-gcp/pulls/${env.CHANGE_ID}", returnStdout: true).trim()
+                        def body = evaluateJson(json,'${json.body}')
+                        if (body.contains("[skip ci]")) {
+                            echo ("'[skip ci]' spotted in PR body text.")
+                            env.shouldBuild = "false"
+                        }
+                    }
+                }
+                sh 'git config --global user.name "upbound-bot"'
+                sh 'echo "machine github.com login upbound-bot password $GITHUB_UPBOUND_BOT" > ~/.netrc'
+            }
+        }
+
+        stage('Build'){
+            when {
+                expression {
+                    return env.shouldBuild != "false"
+                }
+            }
+            steps {
+                sh './build/run make vendor.check'
+                sh './build/run make -j\$(nproc) build.all'
+            }
+            post {
+                always {
+                    archiveArtifacts "_output/lint/**/*"
+                }
+            }
+        }
+
+        stage('Unit Tests') {
+            when {
+                expression {
+                    return env.shouldBuild != "false"
+                }
+            }
+            steps {
+                sh './build/run make -j\$(nproc) test'
+                sh './build/run make -j\$(nproc) cobertura'
+            }
+            post {
+                always {
+                    archiveArtifacts "_output/tests/**/*"
+                    junit "_output/tests/**/unit-tests.xml"
+                    cobertura coberturaReportFile: '_output/tests/**/cobertura-coverage.xml',
+                            classCoverageTargets: '50, 0, 0',
+                            conditionalCoverageTargets: '70, 0, 0',
+                            lineCoverageTargets: '40, 0, 0',
+                            methodCoverageTargets: '30, 0, 0',
+                            packageCoverageTargets: '80, 0, 0',
+                            autoUpdateHealth: false,
+                            autoUpdateStability: false,
+                            enableNewApi: false,
+                            failUnhealthy: false,
+                            failUnstable: false,
+                            maxNumberOfBuilds: 0,
+                            onlyStable: false,
+                            sourceEncoding: 'ASCII',
+                            zoomCoverageChart: false
+                }
+            }
+        }
+
+                stage("Integration Tests"){
+            when {
+                expression {
+                    return env.shouldBuild != "false"
+                }
+            }
+            steps {
+                sh './build/run make -j\$(nproc) e2e'
```

Comment:
Integration test scripts have not been added to this repo, but I can follow up with that. For now, do you want to remove this stage? Example: https://github.com/crossplaneio/stack-gcp/blob/a72c898c8fde9541ed49c5e1e8112e4c92507241/cluster/local/integration_tests.sh